### DISCUSSION TITLE
7.4.6 (Monitor fix and inactive devices)

### DISCUSSION THREAD

**Discussion by T-Troll** [T-Troll/alienfx-tools#236] [Announcements]
###  Changes into this release:

`All light-control tools`:
- Bug fixed: APIv6 (Monitors) interface work incorrectly (doesn't change any lights on device).

`AlienFX-GUI`:
- "Devices and grids" tab enhanced:
  - Inactive (but present into the past) devices also shown at device list
  - Device status string show current device status ("inactive" or API version) and how many physical lights this device have assigned
  - "Clear device" button added - it removes all assigned physical lights from current device, as well as remove it configuration completely if device inactive
  - Other minor UI changes
- "Dimming" settings now apply immediately at "Ambient" tab in ambient mode active.
- Light control subsystem not smarter, in case you try to set the same physical light many times (f.e. it presents in many assigned zones), it only applies last setting, preventing light blink.
- Last selected tab from "Lights" now preserved in case you switch out of "Lights" tab and back.
- If no light devices detected, application now switch to "Settings" tab, not "Fans".
- Bug fixed: Incorrect light brightness calculated into Ambient effect mode.

`AlienFX-Mon`:
- Bug fixed: Memory leaked after monitoring paused and resumed.

Other:
- A lot of small bug fixes and enchantments. Most tools are smaller. Beware of the bugs!

<hr /><em>This discussion was created from the release <a href='https://github.com/T-Troll/alienfx-tools/releases/tag/7.4.6'>7.4.6 (Monitor fix and inactive devices)</a>.</em>

**Comment by Rixtho** [comment]
After a recent update the fans don't spin at all anymore. Even switching profiles doesn't change anything. I already had a bug where all the fan profiles were mixed up and I wasn't able to set a manual fan curve. Since deleting and reinstalling, or updating the app doesn't fix anything for me, I wanted to ask whether there is a way to delete not only the app, but also all stored data from my laptop so I can do a fresh install and try again?

**Reply by T-Troll** [reply]
Yes. There is a small script into application folder named `alienfx-config`. Open CMD and run `alienfx-config delete` - this clear all configuration from the system.

But i don't think this is the source of your issue.

Can you, please, share configuration for me first (open CMD in app folder, and run `alienfx-config backup`, then attach `afx-fans.reg` here)?

**Reply by Rixtho** [reply]
After running `alienfx-config delete` and reinstalling the application my manual profile seems to work again. But the fan profiles are still showing up in the wrong order.

I also had the following error when running `alienfx-config backup`
```
Doing backup...
File afx-lights.reg already exists. Overwrite (Yes/No)?y
The operation completed successfully.
File afx-fans.reg already exists. Overwrite (Yes/No)?y
The operation completed successfully.
File afx-gui.reg already exists. Overwrite (Yes/No)?y
The operation completed successfully.
ERROR: The system was unable to find the specified registry key or value.
Done. 
```

I still got the registration entries you requested though: [afx-reg.zip](https://github.com/T-Troll/alienfx-tools/files/9993957/afx-reg.zip)

**Reply by T-Troll** [reply]
I forgot to mention - all apps should be stopped before delete configuration.
But you don't need to re-install apps later, it's ok to just re-start it.

As i check, Power modes can be out of order. Mode names generated at application start - keeping old one if present and adding new into hardware-specific order.
But after app restart, it will be added in order sorted by power value inside, not info-block order.
So it's normal, it works correct anyway. You can rename it in case you feel not good.

**Reply by Rixtho** [reply]
Okay thanks!
